string[] name
float64[] position
---
bool success